test/run-xspec-tests.cmd

```diff
@@ -64,3 +69,12 @@ rem
 rem Exit as success
 rem
 exit /b 0
+
+:is_schematron
+    set var=%1
+    if not "%var%"=="%var:schematron=%" (

```

Comment:
Would be nice to align it with the shell script: Something like `    if "%var:~0,10%"=="schematron" (`